index.html

```diff
@@ -4,15 +4,246 @@
+            <div class="abt-wrapper">
+                <div class="circle-img"><img src="images/lang.png" alt="Languages"></div>
+                <h3 class="abts-head">Languages</h3>
+                <ul class="works-details">
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                </ul>
+            </div>
+            <div class="abt-wrapper">
+                <div class="circle-img"><img src="images/framew.png" alt="Frameworks"></div>
+                <h3 class="abts-head">Frameworks</h3>
+                <ul class="works-details">
+                    <li>Bootstrap</li>
+                    <li>Ruby on rails</li>
+                    <li>RSpec</li>
+                    <li>Capybara</li>
+                    <li>Selenium</li>
+                </ul>
+            </div>
+    
+            <div class="abt-wrapper">
+                <div class="circle-img"><img src="images/skill.png" alt="Skills"></div>
+                <h3 class="abts-head">Skills</h3>
+                <ul class="works-details">
+                    <li>Codekit</li>
+                    <li>GitHub</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+            </div>  
+        </div>
+        
     </section>
+</div>
+    <!--About content ends Here-->
+
+    <!--Footer content starts Here-->
+    <footer class="footer-container" id="contact-id">
+        <div class="footer-card">
+            <div class="footer-title">
+                <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            </div>
+            
+            <form class="footer-form" action="https://formspree.io/f/myyvbggy" method="POST">
+                <input name="fullname" id="userfullname" class="fulname" type="text" placeholder="Full name" maxlength="30" required>
+                <input name="email" id="useremail" class="email" type="email" placeholder="Email address" required>
+                <textarea name="message1" id="usermessage" class="txt2" cols="30" rows="5" maxlength="500" placeholder="Type something" required></textarea>
+                <h6 class="error_message"></h6>
+                <button class="abt-btn" type="submit">Get in touch</button>
+            </form>
+        </div>
+        <div class="social-link">
+            <ul class="medias">
+                <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fa fa-linkedin"></i></a></li>
+                <li><a href="#"><i class="fa fa-github"></i></a></li>
+                <li><a href="#"><i class="fa fa-medium"></i></a></li>
+                <li><a href="#"><i class="fa fa-angellist"></i></a></li>
+            </ul>
+            <a href="www.gmail.com" class="emai-address">hailubrhanu29@gmail.com</a>
+        </div>
+    </footer>
```

Comment:
- [ OPTIONAL] Kindly remove the contact form as it has not yet part of the requirement :+1: